keyboards/yampad/keymaps/default/keymap.c

```diff
@@ -0,0 +1,142 @@
+#include QMK_KEYBOARD_H
+extern keymap_config_t keymap_config;
```

Comment:
This isn't actually needed, is this is done in another file that is included 
 into this file.
```suggestion
```